pkg/utils/eth.go

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+func GWeiToWei(gwei float64) (*big.Int, error) {
```

Comment:
Let's move those methods to `internal/utils/utils.go`?

Reply:
ok